use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: default
+title: "Beam WordCount Example"
+permalink: /use/wordcount-example/
+redirect_from: /docs/use/wordcount-example/
+---
+
+# Apache Beam WordCount Example
+
+> **Note:** This walkthrough is still in progress. There is an open issue to finish the walkthrough ([BEAM-664](https://issues.apache.org/jira/browse/BEAM-664)).
```

Comment:
Might be nice just to point out explicitly that the tasks remaining (that the reader cares about) are detailed instructions for running across multiple runners.